service/dap/types.go

```diff
@@ -154,9 +154,13 @@ type LaunchAttachCommonConfig struct {
 	ShowRegisters bool `json:"showRegisters,omitempty"`
 
 	// Boolean value to indicate whether system goroutines
-	// should be should be hidden from the call stack view.
+	// should be hidden from the call stack view.
 	HideSystemGoroutines bool `json:"hideSystemGoroutines,omitempty"`
 
+	// String value to indicate which system goroutines
```

Comment:
You might want to link to FILTERING for documentation
https://github.com/go-delve/delve/blob/master/Documentation/cli/README.md#goroutines